src/normalize/normalizer.rs

```diff
@@ -287,7 +277,7 @@ impl Normalizer {
             } else {
                 let binary = Binary {
                     path: entry.path.symbolic_path.to_path_buf(),
-                    build_id: read_build_id(&entry.path.maps_file)?,
+                    build_id: (read_build_id)(&entry.path.maps_file)?,
```

Comment:
very minor and more a curiosity: are these extra `()` needed because of name conflict between argument name and function name? If so, why not change argument to something like `get_build_id`?

Reply:
They are needed because `read_build_id` is now a generic function. That's just the syntax required for calling it then.